docs/specs/20251125-bloom-signaling.md

```
@@ -4,50 +4,60 @@
 - 状態はプロセスメモリに保持し、Bloomプロセス再起動で消える（MVP）。
 
 ## 機能要件
+
 1) ルーム管理
+
 - Room作成: `CreateRoom`要求で新しいroom_idを払い出し、作成者を参加者として登録。
 - 参加/離脱: `JoinRoom`で参加者登録、`LeaveRoom`またはWS切断で離脱扱い。
 - 参加者リスト配信: 参加/離脱時に`RoomParticipants`イベントを全参加者へブロードキャスト。
 - 同時接続上限: MVPでは1 Room あたり最大 8 Participant（ハードコード可）。超過時は`RoomFull`エラーを返す。
 
 2) シグナリング中継（WebSocket）
```

markdownlint-cli2 (0.18.1)

Ordered list item prefix
Expected: 1; Actual: 2; Style: 1/1/1

(MD029, ol-prefix)